test_data/pil/simple_sum_asm_macro.pil

```diff
@@ -0,0 +1,42 @@
+namespace Main(2**10);
+    col witness XInv;
+    col witness XIsZero;
+    XIsZero * (1 - XIsZero) = 0;
+
+    macro if_then_else(condition, true_value, false_value) { condition * true_value + (1 - condition) * false_value };
+    macro jump_to(target) { pc' = target; };
```

Comment:
I know you freak out about the global stuff, but it's much less useful this way. Rewrite syntax looks good, but we need a way to generate multiple statements from a single macro invocation.

Reply:
![image](https://github.com/powdr-org/powdr/assets/504195/a2ca6fa4-8e30-44b8-b943-dcf3127f32bc)

Reply:
>  but it's much less useful this way

I agree with this for instructions, but for macros I think it should be fine, at least in the examples I've seen so far, because they're always used inside instructions so you know what you want to pass to the macro. Moreover the macro `pc` is a parameter anyway.

> we need a way to generate multiple statements from a single macro invocation.

Yea true, then maybe what I suggested isn't the best syntax

Reply:
ok, maybe you are right about the references.

Reply:
although I could imagine references to fixed columns for range constraints or something.